### PR TITLE
add info about send api call

### LINKED ISSUE #615: Add information about api call notifications Callbacks to documentation
What info goes in `$template`? Is that the name of the `api_call_notification`?
Does `data` include `to` and `headers`, etc., or is it just the post data? 
If we are able to set the configuration in `data`, why would `template` be required?

### PR DESCRIPTION
**Pull request by Slashek**
fixes #615

### CODE REVIEW

File: app/views/pages/tutorials/notifications/creating-api-call-notification.liquid [hunk 1]
```diff
@@ -47,13 +50,31 @@ headers: '{
 {% endraw %}
 ```
 
+This defines a POST request to the endpoint with the newly created user's ID. In this example the body of the request is JSON, and to notify the endpoint about it, you set the `Content-Type` header to `application/json`. If the call is successful, then the code from `callback` property is triggered. You can access endpoint's response via [Response](/api-reference/graphql/objects#response) object - which includes headers, status and body. In the example we assume the server returns valid JSON, which we parse and log.
 
-This defines a POST request to the endpoint with the newly created user's ID. The response is not stored, it's fire and forget. In this example the body of the request is JSON, and to notify the endpoint about it, you set the `Content-Type` header to `application/json`.
+### Step 2a: Invoke notification directly via GraphQL [api_call_send](/api-reference/graphql/mutations#api_call_send)
 
-### Step 2: Connect notification to sign up form
+To easily test your newly created Api Call you can trigger it using following query:
```
Comment (pavelloz) [inline app/views/pages/tutorials/notifications/creating-api-call-notification.liquid]:
```suggestion
To easily test your newly created API Call Notification you can trigger it using the following query:
```

File: app/views/pages/tutorials/notifications/creating-api-call-notification.liquid [hunk 2]
```diff
@@ -47,13 +50,31 @@ headers: '{
...
+  ) {
+    response{ status body }
+    errors {
+      message
+    }
+  }
+}
+```
 
+All parameters provided via GraphQL as `data` argument, will be accessible in every Api Call Notification property as a `data` variable (or `form` to make it compatible with FormConfiguration).
```
Comment (pavelloz) [inline app/views/pages/tutorials/notifications/creating-api-call-notification.liquid]:
```suggestion
All parameters provided via GraphQL as `data` argument, will be accessible in every API Call Notification property as a `data` variable (or `form` to make it compatible with FormConfiguration).
```

File: app/views/pages/tutorials/notifications/creating-api-call-notification.liquid [hunk 3]
```diff
@@ -47,13 +50,31 @@ headers: '{
 {% endraw %}
 ```
 
+This defines a POST request to the endpoint with the newly created user's ID. In this example the body of the request is JSON, and to notify the endpoint about it, you set the `Content-Type` header to `application/json`. If the call is successful, then the code from `callback` property is triggered. You can access endpoint's response via [Response](/api-reference/graphql/objects#response) object - which includes headers, status and body. In the example we assume the server returns valid JSON, which we parse and log.
```
Comment (pavelloz) [inline app/views/pages/tutorials/notifications/creating-api-call-notification.liquid]: I think adding link to https://documentation.platformos.com/tutorials/forms/form-server-side-processing might help to propagate this knowledge.

File: app/views/pages/tutorials/notifications/creating-api-call-notification.liquid [hunk 4]
```diff
@@ -47,13 +50,31 @@ headers: '{
...
+To easily test your newly created API Call Notification you can trigger it using the following query:
 
-Connect the API Call Notification you created with the sign up form `form_configurations/developer/sign_up.liquid`:
+```graphql
+mutation send_api_call ($data: HashObject) {
+  api_call_send(
+    data: $data
+    template: { name: "ping_example_com_on_user_sign_up" }
+  ) {
+    response{ status body }
```
Comment (pavelloz) [inline app/views/pages/tutorials/notifications/creating-api-call-notification.liquid]:
```suggestion
    response { status body }
```

**Comment by diana-lakatos** [comment]
Reviewed and edited. Thank you!

**Pull request merged**